server/src/index.ts

```diff
@@ -0,0 +1,86 @@
+import express, { type Request, type Response } from 'express';
+import cors from 'cors';
+import 'dotenv/config';
+
+import { getRecentActivities } from './github.js';
+import { summarizeMarkdown } from './summarize.js';
+import { PostsRepo } from './posts.js';
+
+const app = express();
+app.use(express.json());
+app.use(cors({ origin: process.env.CORS_ORIGIN?.split(',') || '*' }));
```

Comment:
환경변수가 없을 때 '*' 대신 기본값 지정이 안전합니다.

개발환경에서 cors 문제를 회피하고 싶을때는 vite proxy 설정을 찾아봐도 좋을 것 같습니다.

```suggestion
app.use(cors({ 
  origin: process.env.CORS_ORIGIN?.split(',') || ['http://localhost:5173'] 
}));
```